[2025-01-28T08:50:00.131+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-28T08:50:00.147+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_data_ingestion.ingest_data manual__2025-01-28T08:49:56.652598+00:00 [queued]>
[2025-01-28T08:50:00.153+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_data_ingestion.ingest_data manual__2025-01-28T08:49:56.652598+00:00 [queued]>
[2025-01-28T08:50:00.153+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-28T08:50:00.163+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): ingest_data> on 2025-01-28 08:49:56.652598+00:00
[2025-01-28T08:50:00.172+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_data_ingestion', 'ingest_data', 'manual__2025-01-28T08:49:56.652598+00:00', '--job-id', '44', '--raw', '--subdir', 'DAGS_FOLDER/dag_data_ingestion.py', '--cfg-path', '/tmp/tmp42nbcexm']
[2025-01-28T08:50:00.174+0000] {standard_task_runner.py:105} INFO - Job 44: Subtask ingest_data
[2025-01-28T08:50:00.371+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=188) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-01-28T08:50:00.371+0000] {standard_task_runner.py:72} INFO - Started process 190 to run task
[2025-01-28T08:50:00.397+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_data_ingestion.ingest_data manual__2025-01-28T08:49:56.652598+00:00 [running]> on host 7537dc2fd5ca
[2025-01-28T08:50:00.461+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_data_ingestion' AIRFLOW_CTX_TASK_ID='ingest_data' AIRFLOW_CTX_EXECUTION_DATE='2025-01-28T08:49:56.652598+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-28T08:49:56.652598+00:00'
[2025-01-28T08:50:00.461+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-28T08:50:00.471+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-01-28T08:50:00.472+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Hello World!"']
[2025-01-28T08:50:00.485+0000] {subprocess.py:99} INFO - Output:
[2025-01-28T08:50:00.487+0000] {subprocess.py:106} INFO - Hello World!
[2025-01-28T08:50:00.487+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-01-28T08:50:00.510+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-28T08:50:00.510+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_data_ingestion, task_id=ingest_data, run_id=manual__2025-01-28T08:49:56.652598+00:00, execution_date=20250128T084956, start_date=20250128T085000, end_date=20250128T085000
[2025-01-28T08:50:00.545+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-28T08:50:00.558+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-28T08:50:00.559+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
